work-locally.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Work Locally
-description: "Seamless local-to-cloud experience for rapid experimentation"
+description: "Run your Modus app locally on your machine using the Hyp CLI"
 ---
 
 Building Modus apps is even easier when you add Hypermode. The Hyp CLI extends
```

Comment:
@ryanfoxtyler The opening lines sound okay from a more marketing standpoint, but I would love to simplify this down to make it easier to understand. What content/terms from this would you say are a "must"?

Full content: `Building Modus apps is even easier when you add Hypermode. The Hyp CLI extends the [local Modus experience](https://docs.hypermode.com/modus/modus-cli) with access to Hypermode-hosted models defined in your [app’s manifest](https://docs.hypermode.com/modus/app-manifest) for rapid experimentation with new AI models.`

Reply:
I think the key is accessing models when running your Modus app locally (but the Modus CLI is responsible for the running of that app locally)

Reply:
Hmmm maybe something like: `When you are building Modus apps, you can use the Hyp CLI locally to access Hypermode-hosted models defined in your app's manifest.`

🤔

Reply:
That feels really long for a description?

Reply:
I'm fine with simplifying!

`When you are building Modus apps, you can use the Hyp CLI to access Hypermode models.`

or even `You can use the Hyp CLI to access Hypermode models.`